style.css

```diff
@@ -0,0 +1,846 @@
+
+.bottom-logos img {
+  height: 40px;
+  object-fit: contain;
+  mix-blend-mode: color-burn;
+  margin: 15px;
+}
+
+/* Footer Section */
+.footer {
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 20px;
+}
+
+.footer img {
+  height: 80px;
+}
+
+.footer-text {
+  display: flex;
+  flex-direction: column;
+}
+
+@media screen and (min-width: 768px) {
+  .partner {
+    display: block;
+    text-align: center;
+    background-color: #534f4e;
+    padding: 40px 10px;
+  }
+
+  .partner-header {
+    margin-bottom: 5px;
+    color: white;
+  }
+
+  .bottom-logos {
+    flex-direction: row;
+    justify-content: center;
+    padding: 10px 20px;
+    width: auto;
+  }
+
+  .bottom-logos img {
+    height: 50px;
+    margin: 0 30px;
+  }
+
+  .footer {
+    display: flex;
+    justify-content: space-between;
+    padding: 40px 20%;
+    align-items: center;
+    gap: 23px;
+  }
+
+  .footer img {
+    height: 3rem;
+  }
+
+  .footer-about {
+    margin-top: 10px;
+  }
+
+  .footer-text {
+    display: flex;
+    flex-direction: column;
+    font-size: 11px;
+  }
+}
+
+/* =====================================About page=============================================== */
+
+.top-h1-about {
+  color: #ec5242;
+  font-size: 20px;
+  margin-bottom: 10px;
+}
+
+.main-par-about {
+  color: #272a31;
+  font-size: 16px;
+  text-align: center;
+  background-color: white;
+  padding: 20px;
+  margin: 10px 20px;
+}
+
+/* Partner Section */
+
+.email-id {
+  margin-top: 10px;
+  text-decoration: underline;
+  font-weight: bold;
+}
+
+.past-projects {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  margin: 20px auto;
+  height: 150px;
+  width: 90%;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.project-1 {
+  background-image: url("./images/about_2013.png");
+}
+
+.project-2 {
+  background-image: url("./images/about_2011.png");
+}
+
+.center-about {
+  text-align: center;
+}
+
+.about-h2 {
+  text-align: center;
+  margin: 10px 10px;
+}
+
+.about-par {
+  text-align: center;
+  margin: 10px 30px;
+}
+
+.section-2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-bottom: 1px solid rgb(223, 223, 223);
+}
+
+.section-3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.join-btn {
+  width: 80%;
+}
+
+.about-logo {
+  width: 90%;
+  margin: 10px auto;
+}
+
+.about-footer {
+  display: grid;
+  grid-template-columns: 60% 40%;
+}
+
+.footer-image {
+  background-image: url('./images/main-footer.png');
+  height: 75px;
+  width: 156px;
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.footer-copyright {
+  font-size: x-small;
+}
+
+.footer-about-text {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .about-container {
+    background: url(./images/about_bg_01.png);
+  }
```

Comment:
- [ ] As suggested by the previous reviewer, consider adding `background-color: #f7f7f7`to .about-container on all screen sizes to have light gray background-color in the about section.